src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -20,15 +20,37 @@ internal abstract partial class ConversionsBase
 
         internal readonly bool IncludeNullability;
 
-        protected ConversionsBase(AssemblySymbol corLibrary, int currentRecursionDepth, bool includeNullability)
+        private ConversionsBase _lazyOtherNullability;
+
+        protected ConversionsBase(AssemblySymbol corLibrary, int currentRecursionDepth, bool includeNullability, ConversionsBase otherNullabilityOpt)
```

Comment:
>ConversionsBase [](start = 18, length = 15)

xml doc would be useful

Reply:
Added comments to field and `WithNullability`.

---
In reply to: [207374885](https://github.com/dotnet/roslyn/pull/28959#discussion_r207374885) [](ancestors = 207374885)